movie-app/src/Components/MovieCard.js

```diff
@@ -1,14 +1,34 @@
+
+import {useState} from 'react';
+
 const MovieCard = ({movie}) => {
+  const moviePoster = {
+    backgroundImage: `url(${movie.Poster})`,
+  }
+
+  const [cardHover, setCardHover] = useState(false);
+// when true, apply show info animation
```

Comment:
comment here

Reply:
This is a documentation comment.  Those are great!  What we don't want is old commented out code hanging around in our files that we don't plan to use.